Taskfile.yml

```diff
@@ -109,6 +110,7 @@ tasks:
           DATA_DIR: "{{.OUTPUT_DIR}}"
           OUTPUT_FILE: "{{.CHECKSUM_FILE}}"
     sources:
+      - "{{.G_BUILD_DIR}}/meteor.md5"
```

Comment:
```suggestion
```

Since meteor is not used directly in this task, I think it's better to rely on `webui` to change when `meteor` changes.